### ISSUE TITLE
Split large queries into chunks; run each in parallel

### ISSUE THREAD

**Issue by ewaters** [ewaters/altsql-shell#8]
@gt8073a suggests a feature where one could take a large query which will be fairly resource intensive on the server side and find a way to split it up into smaller chunks and run each in parallel in the background, waiting for the results and then putting them together in such a way as to make it transparent to the user.
